omero/autogen_docs

```diff
@@ -34,9 +35,10 @@ sed 1,5d advanced-help.txt > omero/downloads/inplace/advanced-help.txt
 omero import --javahelp 2> java-help.txt || echo "Dumped"
 sed 1,5d java-help.txt > omero/downloads/cli/help.out
 
-echo "Generating DB script example"
-omero db script --password secretpassword 2>&1 | sed "s|Saving to.*/|Saving to /home/omero/|" > omero/downloads/cli/db-script-example.txt
-rm OMERO*sql
+#echo "Generating DB script example"
```

Comment:
Was this supposed to be protected by an `if DB` statement?

Reply:
This was commented out during testing and I forgot about it
The workflow will be a bit more complex for that section to work but since we are planning to move to rtd, we need to  everything updated via action. I will push extra commits to sort this section

Reply:
This is now back on see https://github.com/jburel/ome-documentation/actions/runs/1529949753
This requires a more complex setup in the action unfortunately see a024358